### DISCUSSION TITLE
Daily Load includes essential but not non-essential

### DISCUSSION THREAD

**Discussion by rickyrickuk** [slipx06/sunsynk-power-flow-card#373] [General]
Hi
I have all my load on the non essentials side, with a double socket for emergencies connected to the essentials connection on my Sunsynk inverter.
The Daily Load seems to only include the essentials total, is it possible to have the Daily Load to be essential + non essential loads?
Thanks

**Comment by slipx06** [comment]
Hi. You will need to create your own template sensor that sums this and use it as your day load energy. The card does not perform any calculations for daily energy apart for some auto scaling.
